### ISSUE TITLE
does not exist on type 'typeof global'

### ISSUE THREAD

**Issue by joma74** [Microsoft/TypeScript#27057]
**TypeScript Version:** 3.1.0-dev.20180912

**Search Terms:**
js ts global nodejs

**Code**
See Playground Link

**Expected behavior:**
No tsc error

**Actual behavior:**
![tsglobalissue-2018-09-12_21 15 19](https://user-images.githubusercontent.com/5314859/45450801-90e6bf80-b6d9-11e8-9fa8-a0e9c30437f2.gif)

`[ts] Property 'getErrorAsString' does not exist on type 'typeof global'.`

A second - possibly related issue - is that the declaration of `clazz1bGlobalVar: clazz1b`  in `./types/node-global.d.ts` is infered to `(property) NodeJS.Global.clazz1bGlobalVar: any`

A third - possibly related issue - is that if you alter the declaration of `clazz1bGlobalVar: clazz1b` in `./types/node-global.d.ts` to `clazz1bGlobalVar: number`, it is infered correctly in this file as `(property) NodeJS.Global.clazz1bGlobalVar: number`. But in e.g. `index.js`, you can assign just a value of any type to that, e.g. `global.clazz1bGlobalVar= "clazz1bInstance"`. And on hover `(property) global.clazz1bGlobalVar: string` is shown.

**Playground Link:**
```
git clone https://github.com/joma74/ts-in-js-1.git
git checkout ts-issue-global-demo
cd ts-in-js-1
yarn install
yarn tsc_validate_watch
```
Open ```index.js``` and change as shown in video

**Related Issues:**
None found

**Comment by joma74** [comment]
This should already be addressed by `types/node-global.d.ts` in the playground link. If not, sorry, i could not follow your advice.

**Comment by brendankenny** [comment]
I believe this was fixed by #26918 (like #27099 was) and should be good to go with `typescript@next`

**Comment by joma74** [comment]
@brendankenny Thanks for the heads-up!
Confirm that this is fixed in 3.1.0-dev.20180919
